recipes/pscript/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pscript" %}
+{% set version = "0.5.3" %}
+{% set sha256 = "cabd070f47329d5e7b23cb479cced5da38492262d742bca9861383a06d71522c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
We recommend [using `pip`]( https://conda-forge.org/docs/meta.html#use-pip ) now. Please update accordingly.